src/main/scala/gitbucket/core/servlet/GitRepositoryServlet.scala

```diff
@@ -523,6 +512,51 @@ class WikiCommitHook(owner: String, repository: String, pusher: String, baseUrl:
     }
   }
 
+  private[this] def buildActionInfo(
+    commit: JGitUtil.CommitInfo,
+    action: String,
+    fileName: String
+  ): Option[WikiActivityActionInfo] = {
+    action match {
```

Comment:
We may be able to use `ChangeType` directly for this pattern matching instead of the mapped string.

Reply:
I used the mapped string here because we already use it downstream as a parameter to another function call.

Reply:
Oh, I see. Thanks for clarification.